config.default.js

```diff
@@ -52,38 +46,14 @@ function getFileEnv(envVariable) {
   return origVar;
 }
 
-const meConfigMongodbServer = process.env.ME_CONFIG_MONGODB_SERVER
-  ? process.env.ME_CONFIG_MONGODB_SERVER.split(',')
-  : false;
-
-function getConnectionStringFromEnvVariables() {
```

Comment:
With this change, even if connection string is recommended, we lost the way of set individual config changes, and this would be a breaking change for people that use like that. This also should add some deprecation warning instead and not break the change until later versions

Reply:
We can say something like "in v2.0.0 this configuration option will no longer be available". Let's give users ample time to actually notice the warning and update their configuration. At least we can say we tried when complaints about how the config people have been using for years stopped working. :)